src/compiler.js

```diff
@@ -67,6 +67,7 @@ if (settingsFile) {
 EXPORTED_FUNCTIONS = set(EXPORTED_FUNCTIONS);
 EXCEPTION_CATCHING_ALLOWED = set(EXCEPTION_CATCHING_ALLOWED);
 IMPLEMENTED_FUNCTIONS = set(IMPLEMENTED_FUNCTIONS);
+SIDE_MODULE_SYMBOLS = set(SIDE_MODULE_SYMBOLS);
```

Comment:
There is a mix of terminology here, e.g. implemented vs exported (vs defined, which is probably the "correct" thing as it applies to linking symbols), as well as functions vs symbols, etc. Are functions and symbols equivalent in this context? Are exported and defined equivalent in this context?

Reply:
I have another PR that renames `IMPLEMENTED_FUNCTIONS` to `WASM_EXPORTS` so you are right we should probably call this `SIDE_MODULE_EXPORTS` which is a more accurate description.

Reply:
If we can land this change first then the names won't appear so different: https://github.com/emscripten-core/emscripten/pull/13917